### PR TITLE
Feat: new version available message on android and ios

### PR DESCRIPTION
**Pull request by rastajpa**

### CODE REVIEW

**Comment by cmgustavo** [comment]
I mark it as "requested changes" just to review it a little bit more.

File: src/pages/home/home.spec.ts [hunk 1]
```diff
@@ -5,13 +5,29 @@ import { Subject } from 'rxjs';
...
+import { AddPage } from '../add/add';
+import { CopayersPage } from '../add/copayers/copayers';
+import { WalletDetailsPage } from '../wallet-details/wallet-details';
 import { ConfigProvider } from './../../providers/config/config';
+import { ActivityPage } from './activity/activity';
 import { HomePage } from './home';
+import { ProposalsPage } from './proposals/proposals';
 
-describe('HomePage', () => {
+fdescribe('HomePage', () => {
```
Comment (bitjson) [inline src/pages/home/home.spec.ts]: Looks like an `fdescribe` was left in here

File: src/pages/home/home.ts [hunk 2]
```diff
@@ -405,29 +408,66 @@ export class HomePage {
...
+          }
+          this.showAvailableVersionCard(version);
+        }
+      )
+      .catch(err => {
+        this.logger.error('Error getLatestAppVersion', err);
+      });
+  }
+
+  private showAvailableVersionCard(version) {
```
Comment (bitjson) [inline src/pages/home/home.ts]: Could you add the types of methods? `version` here is a `string` right? (Ideally we can get real types for everything, without using `any`)

**Comment by bitjson** [comment]
(And agree with @cmgustavo 👍)

File: src/pages/home/home.spec.ts (hunk 1 above)
Comment (rastajpa) [reply src/pages/home/home.spec.ts]: fixed up

**Pull request closed without merging**